### PR TITLE
proxy: Extend log record format

### PR DESCRIPTION
**Pull request by rlenglet**
Signed-off-by: Romain Lenglet <romain@covalent.io>

### CODE REVIEW

**Review by aalemayhu — APPROVED** [review]
LGTM, build failure is unrelated and fixed in #1435.

File: pkg/proxy/accesslog/record.go [hunk 1]
```diff
@@ -65,17 +68,43 @@ const (
...
-	IPv6     string
-	Port     uint16
-	Identity uint64
-	Labels   []string
+	ID          uint64
+	IPv4        string
+	IPv6        string
+	Port        uint16
+	Identity    uint64
+	LabelSHA256 string
```
Comment (tgraf) [inline pkg/proxy/accesslog/record.go]: I think @manalibhutiyani renamed this to `LabelsSHA256` now.

**Review by tgraf — CHANGES_REQUESTED** [review]

**Review by aanm — APPROVED** [review]

File: pkg/proxy/accesslog/record.go (hunk 1 above)
Comment (manalibhutiyani) [reply pkg/proxy/accesslog/record.go]: As per Andre's review comments, yes he wanted me to change this to LabelsSHA256. Will be pushing that  out in some time.

Comment (rlenglet) [reply pkg/proxy/accesslog/record.go]: Done.

**Review by tgraf — APPROVED** [review]

**Comment by tgraf** [comment]
PR #1431 has introduced more uses of the request field in the log structure. So this needs to be rebased/fixes to account for that.

```
21:55:36 ==> cilium-master-build-PR-1434-6: # github.com/cilium/cilium/pkg/proxy/accesslog
21:55:36 ==> cilium-master-build-PR-1434-6: ../pkg/proxy/accesslog/log.go:118: l.request undefined (type *LogRecord has no field or method request, but does have Request)
21:55:36 ==> cilium-master-build-PR-1434-6: ../pkg/proxy/accesslog/log.go:119: l.request undefined (type *LogRecord has no field or method request, but does have Request)
21:55:36 ==> cilium-master-build-PR-1434-6: ../pkg/proxy/accesslog/log.go:120: l.request undefined (type *LogRecord has no field or method request, but does have Request)
21:55:36 ==> cilium-master-build-PR-1434-6: ../pkg/proxy/accesslog/log.go:121: l.request undefined (type *LogRecord has no field or method request, but does have Request)
```

**Review by tgraf — DISMISSED** [review]
CI failure

**Review by manalibhutiyani — APPROVED** [review]

**Comment by rlenglet** [comment]
Rebased on master after #1425 got merged.

**Review by michi-covalent — APPROVED** [review]

**Pull request merged**